### PR TITLE
Snackbar update

### LINKED ISSUE #1660: Snackbar notification update
### Description

When there is more than 1 line of text, layout of this component breaks. 

### Figma link

https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-Components?node-id=216%3A2717

### How urgent this is?

Not a blocker, but still worth considering

### PR DESCRIPTION
**Pull request by drillprop**
Fix #1660 and #2005
Design: https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-Components?node-id=4606%3A140879

### CODE REVIEW

**Comment by drillprop** [comment]
If I'm not mistaken designers are still working on the look of this component: https://github.com/Joystream/atlas/issues/2005
I'll set this as a draft for now.

**Comment by drillprop** [comment]
The designs were just updated by Adam. I'll mark this as ready for dev when I'm done.

File: packages/atlas/src/components/Snackbar/Snackbar.tsx [hunk 1]
```diff
@@ -1,40 +1,41 @@
...
   SnackbarDescription,
-  SnackbarHeader,
   SnackbarIconContainer,
-  SnackbarTitle,
   SnackbarWrapper,
   StyledInnerWrapper,
 } from './Snackbar.styles'
 
-export type SnackbarVariant = 'primary' | 'secondary'
+import { Text } from '../Text'
```
Comment (rafalpawlow) [inline packages/atlas/src/components/Snackbar/Snackbar.tsx]: Let's use absolute import

File: packages/atlas/src/providers/uploadsManager/uploadsManager.tsx [hunk 2]
```diff
@@ -70,8 +69,7 @@ export const UploadsManager: React.FC = () => {
             description: video.parentObject?.title || '',
             iconType: 'success',
             timeout: UPLOADED_SNACKBAR_TIMEOUT,
-            actionText: 'See on Joystream',
-            actionIcon: <SvgActionNewTab />,
+            actionText: 'Watch',
```
Comment (rafalpawlow) [inline packages/atlas/src/providers/uploadsManager/uploadsManager.tsx]: Can you provide source of this change? I cannot find it in figma

**Review by rafalpawlow — CHANGES_REQUESTED** [review]

File: packages/atlas/src/providers/uploadsManager/uploadsManager.tsx (hunk 2 above)
Comment (drillprop) [reply packages/atlas/src/providers/uploadsManager/uploadsManager.tsx]: In the first version of the snack bar, the action button was placed on the right, close to the close button and there wasn't a lot of space for longer text and icons. That's why I initially changed this. 
This is how designers wanted snakcbars to look initially: 
image

Anyway, they changed the design again in #2005, so I'll just revert this change ;) 
image

**Review by rafalpawlow — APPROVED** [review]

**Pull request merged**